otherlibs/stdune/src/fpath.ml

```diff
@@ -217,3 +217,11 @@ let traverse =
 let traverse_files ~dir ~init ~f =
   traverse ~dir ~init ~on_dir:(fun ~dir:_ _fname acc -> acc) ~on_file:f
 ;;
+
+let is_rw p =
+  try
+    Unix.access p Unix.[ R_OK; W_OK ];
+    true
+  with
+  | Unix.Unix_error _ -> false
```

Comment:
I think

```suggestion
  match Unix.access p Unix.[ R_OK; W_OK ] with
  | () -> true
  | exception Unix.Unix_error _ -> false
```

reads a little better as it separates the thing that might throw from the value it will compute, but I also just really like `match`.

Reply:
I also love the `match` syntax more than the `try` :)